src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+
+  sourceString.split(';').forEach((declaration) => {
+    let [property, value] = declaration.split(':');
+
+    if (property && value) {
+      property = property.trim();
+      value = value.trim();
+      styles[property] = value;
+    }
+  });
```

Comment:
[CODE KNOWLEDGE] - Don't use iteration methods to modify an already existing array/object. While `forEach` is an exception to this rule, consider using `reduce` here to directly construct the `styles` object from the array of declarations.